src/database/queries/userChangemakerPermissions/selectByPrimaryKey.sql

```diff
@@ -0,0 +1,5 @@
+SELECT user_changemaker_permission_to_json(user_changemaker_permissions.*) AS "object"
+FROM user_changemaker_permissions
+WHERE user_keycloak_user_id = :userKeycloakUserId
+  AND changemaker_id = :changemakerId
+  AND permission = :permission
```

Comment:
Do we want a `AND (not_after > now() OR not_after IS NULL`? Or do we also want to expose "deleted" rows? Are we using this outside of tests yet?

Reply:
Per the other conversation this is now updated so `not_after` is purely an internal concept to the DB; it's not exposed to node at all and any expired record is treated as if it doesn't exist for all queries.